vectornav/src/vn_sensor_msgs.cc

```diff
@@ -114,6 +114,22 @@ class VnSensorMsgs : public rclcpp::Node
   }
 
 private:
+  void convert_to_enu(const vectornav_msgs::msg::CommonGroup::SharedPtr msg_in, sensor_msgs::msg::Imu msg_out) const
```

Comment:
This does not take the output message by reference, and so none of the fields are actually set.
```suggestion
  void convert_to_enu(const vectornav_msgs::msg::CommonGroup::SharedPtr msg_in, sensor_msgs::msg::Imu &msg_out) const
```

Reply:
Otherwise I can confirm this works after testing on a real robot.